kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/OperatorChangeDetectionST.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests;
+
+import java.util.List;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.skodjob.testframe.resources.KubeResourceManager;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxyIngress;
+import io.kroxylicious.kubernetes.operator.assertj.OperatorAssertions;
+import io.kroxylicious.systemtests.installation.kroxylicious.Kroxylicious;
+import io.kroxylicious.systemtests.installation.kroxylicious.KroxyliciousOperator;
+import io.kroxylicious.systemtests.k8s.KubeClient;
+
+import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assumptions.assumeThat;
+
+/**
+ * The Kroxylicious system tests.
+ */
+class OperatorChangeDetectionST extends AbstractST {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(OperatorChangeDetectionST.class);
```

Comment:
Thanks for trailing blazing here, we'd will be strong as a project if the ST are a shared concern.

Reply:
Yeah I despise the trap we have fallen into of sending out a flare to get system tests written. 

I'm not yet at the point of requiring them for a merge but thats the direction I'd like to go